AdyenDropIn/DropInComponent.swift

```diff
@@ -68,15 +74,68 @@ public final class DropInComponent: NSObject, PresentableComponent {
     public func handle(_ action: Action) {
         actionComponent.handle(action)
     }
+
+    // MARK: - Handling Partial Payments
+
+    /// :nodoc:
+    private lazy var apiClient: APIClientProtocol = {
```

Comment:
Maybe we can make another `manager` object or somehow move this logic outside?
DropIn already almost 400 lines of code..